src/services/mcp/McpHub.ts

```diff
@@ -343,10 +323,6 @@ export class McpHub {
 		const connection = this.connections.find((conn) => conn.server.name === name)
 		if (connection) {
 			try {
-				// connection.client.removeNotificationHandler("notifications/tools/list_changed")
-				// connection.client.removeNotificationHandler("notifications/resources/list_changed")
-				// connection.client.removeNotificationHandler("notifications/stderr")
-				// connection.client.removeNotificationHandler("notifications/stderr")
```

Comment:
@celestial-vault, same here. Please explain why we're keeping this or otherwise remove

Reply:
> @celestial-vault, same here. Please explain why we're keeping this or otherwise remove

these are being removed though..